pkg/actors/actors.go

```diff
@@ -514,6 +548,14 @@ func (a *actorsRuntime) callLocalActor(ctx context.Context, req *invokev1.Invoke
 	return resp, nil
 }
 
+func (a *actorsRuntime) getAppChannel(actorType string) channel.AppChannel {
+	if a.internalAppChannel.Contains(actorType) {
+		return a.internalAppChannel
+	} else {
+		return a.externalAppChannel
```

Comment:
nit: typically this else is omitted following the go code style, so you can return straight

source: https://go.dev/doc/effective_go#if